package/scripts/prmon_plot.py

```diff
@@ -193,24 +252,20 @@ def main():
     parser.set_defaults(diff=False)
     args = parser.parse_args()
 
-    # Check the input file exists
-    if not os.path.exists(args.input):
-        print(f"ERROR:: Input file {args.input} does not exists")
-        sys.exit(-1)
-
-    # Load the data
-    data = pd.read_csv(args.input, sep="\t")
-    data["Time"] = pd.to_datetime(data["Time"], unit="s")
+    inputs = args.input.split(",")
+    ylist = args.yvar.split(",")
+    data = []
 
-    # Check the variables are in data
-    if args.xvar not in list(data):
-        print(f"ERROR:: Variable {args.xvar} is not available in data")
+    if len(inputs) > 7:
```

Comment:
Why `7`? I think we can drop this limitation and let the user decide if the final plot looks overcrowded or not.

Reply:
We have removed the limit, although the reason for 7 was that there was only 7 line types.